pkg/eval/builtin_special_test.go

```diff
@@ -24,6 +24,9 @@ func TestVar(t *testing.T) {
 		That("var 'a/b'", "put $'a/b'").Puts(nil),
 		// Declaring one variable whose name ends in ":".
 		That("var a:").DoesNothing(),
+		// Declaring a variable whose name ends in "~" has default value $builtin:nop rather than
+		// $builtin:nil. See https://github.com/elves/elvish/issues/1248.
```

Comment:
No need to refer to the old behavior here, just say that it's equivalent to the `nop` function